orchestrate.py

```diff
@@ -118,6 +121,15 @@ def get(self, path=None):
                 yield self.pool.adhoc(user)
 
                 url = path
+            else:
+                # There is no path or it represents a subpath of the notebook server
+                # Assign a prelaunched container from the pool and redirect to it.
+                container_path = self.pool.acquire().path
+                app_log.info("Allocated [%s] from the pool.", container_path)
+
+                # If no path is set, append self.redirect_uri to the redirect target, else
+                # redirect to specified path.
+                url = "/{}/{}".format(container_path, self.redirect_uri if path is None else path.lstrip('/'))
```

Comment:
Can you try converting this into an `if-else` block instead of using an inline `if-else`?